skipper_test.go

```diff
@@ -186,6 +186,14 @@ func TestOptionsTLSConfig(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, uint16(tls.VersionTLS13), c.MinVersion)
 	require.Equal(t, []tls.Certificate{cert, cert2}, c.Certificates)
+
+	// TLS Cipher Suites
+	o = &Options{CipherSuites: []uint16{1}}
+	c, err = o.tlsConfig(cr)
+	require.NoError(t, err)
+	require.NotEqual(t, len(c.CipherSuites), nil)
```

Comment:
minor: this is redundant or should be `require.NotNil`.
Also I think it should be `require.NoError` and then `assert.Equal(t, len(c.CipherSuites), 1)` - require preconditions and assert expectations.